### ISSUE TITLE
1.1.1 Non-text Content

### ISSUE THREAD

**Issue by ymdahi** [LibreTexts/Libretext#202]
- [1.1.1 Non-text Content](http://www.w3.org/TR/WCAG20/#text-equiv-all)	
- Conformnace Level: Partially Supports

### Remarks and Explanations
Meaningful images do not have alt-text descriptions.  Examples: the alt-text for the image of the LibreTexts chemistry logo contains a file name; the Single Sign-On icon is not recognizable with assistive technology; the alt-text for the PDF image is not descriptive (picture_as_pdf); the Readability and Cite this page icons are not recognizable with assistive technology; the general overview of the LibreVerse image does not have descriptive alt-text and the alt-text for the image contains a file name; and except for the Merlot logo, the alt-text for the LibreTexts supporters' logos are acronyms without including the text in the images.

Sidebar examples: The alt-text descriptions for the periodic table does not include the information associated with the options from the Property to Display dropdown box, such as electron configuration.  Meaningful images on the Sidebar References Organic Chemistry Glossary pages do not have alt-text or have file names as alt-text. 

Decorative images do not have alt-text descriptions. 

When background images that represent meaningful information are used, the information is presented as an accessible alternative. 

CAPTCHA images are not used on the pages.

### Potential fixes/tasks

- [ ] generate report of non-conformant images/figures in libraries
- make the following fixes:
  - [x] add alt text to logo in chem lib
  - [x] the Single Sign-On icon is not recognizable with assistive technology
  - [ ] the alt-text for the PDF image is not descriptive (picture_as_pdf)
  - [ ] the Readability and Cite this page icons are not recognizable with assistive technology
  - [ ] the general overview of the LibreVerse image does not have descriptive alt-text and the alt-text for the image contains a file name
  - [ ] except for the Merlot logo, the alt-text for the LibreTexts supporters' logos are acronyms without including the text in the images
  - [ ] alt-text descriptions for the periodic table does not include the information associated with the options from the Property to Display dropdown box, such as electron configuration
  - [ ] Meaningful images on the Sidebar References Organic Chemistry Glossary pages do not have alt-text or have file names as alt-text
